builtins/amp-video.js

```diff
@@ -49,24 +69,23 @@ export function installVideo(win) {
         assertHttpsUrl(this.element.getAttribute('src'), this.element);
       }
       this.propagateAttributes(
-          ['src', 'controls', 'autoplay', 'muted', 'loop', 'poster'],
-          video);
-      video.width = getLengthNumeral(width);
-      video.height = getLengthNumeral(height);
-      this.applyFillContent(video, true);
+          ['src', 'controls', 'autoplay', 'muted', 'loop', 'preload'],
```

Comment:
It looks like we need to reset `preload`. E.g. above we set it to `none` and if it wasn't defined, it will NOT be set back to the expected default value here. Thus we should move it out of `propagateAttributes` and set it directly with the correct default.